src/DataGrid.tsx

```diff
@@ -332,13 +418,114 @@ function DataGrid<R, K extends keyof R, SR>({
     onColumnResize?.(column.idx, width);
   }, [columnWidths, onColumnResize]);
 
-  function handleRowsUpdate(event: RowsUpdateEvent) {
-    onRowsUpdate?.(event);
+  function handleCommit({ cellKey, rowIdx, updated }: CommitEvent) {
+    onRowsUpdate?.({
+      cellKey,
+      fromRow: rowIdx,
+      toRow: rowIdx,
+      updated,
+      action: UpdateActions.CELL_UPDATE
+    });
+
+    setSelectedPosition(({ idx, rowIdx }) => ({ idx, rowIdx, mode: 'SELECT' }));
```

Comment:
Why not just using spread operator, I thought that would be more concise and we can avoid same variable name though they are in different scope?

```javascript
    setSelectedPosition(position => ({ ...position, mode: 'SELECT' }));
```

Reply:
`selectedPosition` also has a `key` field that we do not want to copy